src/top-bar/workflow-select/workflow-select-option.js

```diff
@@ -0,0 +1,33 @@
+import { MenuItem } from '@dhis2/ui'
+import PropTypes from 'prop-types'
+import React from 'react'
+import classes from './workflow-select-option.module.css'
+
+const WorkflowSelectOption = ({ name, periodType, active, onClick }) => {
+    const label = (
+        <div className={classes.option}>
+            <span className={classes.title}>{name}</span>
+            <span className={classes.subtitle}>
+                {periodType || 'UNAVAILABLE'}
+            </span>
+        </div>
+    )
+
+    return (
+        <MenuItem
+            className={classes.bordered}
+            active={active}
+            onClick={onClick}
+            label={label}
+        />
```

Comment:
`href` to make it accessible

Reply:
I'm deferring this for now, see [DHIS2-11426](https://jira.dhis2.org/browse/DHIS2-11426).